krakend/tests/conftest.py

```diff
@@ -43,7 +43,18 @@ def dd_environment(dd_save_state):
     ):
         asyncio.run(generate_sample_traffic())
 
-        yield {"instances": [{"openmetrics_endpoint": OPEN_METRICS_ENDPOINT}]}
+        yield (
+            {
+                "instances": [{"openmetrics_endpoint": OPEN_METRICS_ENDPOINT}],
+            },
+            {
+                "docker_volumes": [
+                    "/var/run/docker.sock:/var/run/docker.sock:ro",
+                    "/var/lib/docker/containers:/var/lib/docker/containers:ro",
+                    "/opt/datadog-agent/run:/opt/datadog-agent/run:rw",
```

Comment:
why is this needed?

Reply:
I thought about that but since the start command already mounts some for Windows (not sure why) I thought this might not be a big deal since the only impact is that e2e tests would have those loaded.

If we are worried about having these loaded in E2E then I would probably go with the previous approach I had which is loading an entirely different dd_environment for the lab environment (including the docker file to avoid having logs labels added to the containers as well). Not sure if those labels would have an impact on the agent configuration when loading E2E. Will up date this, thanks!

Reply:
awesome, thank you!